content/workshops/How to be a great Mentor/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: How To Be A Great Mentor
+---
```

Comment:
please add the line "ready: true" between "title:" and "---"otherwise it shows up as a [TODO] when we use the content link.